src/content/services/monetizationTagManager.ts

```diff
@@ -180,9 +180,7 @@ export class MonetizationTagManager extends EventEmitter {
 
         handledTags.add(target)
       } else if (hasTarget && !typeSpecified) {
-        const stopMonetizationTag = this.onRemovedTag(target, true)
-        stopMonetizationTags.push(stopMonetizationTag)
-
+        stopMonetizationTags.push(this.onRemovedTag(target))
```

Comment:
Just for a clearer diff:
```suggestion
        const stopMonetizationTag = this.onRemovedTag(target)
        stopMonetizationTags.push(stopMonetizationTag)

```